tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32
+envlist = py{26,27,32}
```

Comment:
This change must do done in a separated commit and you must require the minimum tox version supporting {...} syntax. Sorry, I don't remember the minimum version.